gradle.properties

```diff
@@ -2,24 +2,50 @@
 # Copyright © 2014-2020 The Android Password Store Authors. All Rights Reserved.
 # SPDX-License-Identifier: GPL-3.0-only
 #
+org.gradle.parallel=true
+org.gradle.configureondemand=true
+org.gradle.caching=true
 
-# AndroidX
-android.enableJetifier=false
-android.useAndroidX=true
+# Enabling filesystem watching
+org.gradle.vfs.watch=true
+org.gradle.unsafe.watch-fs=true
```

Comment:
First one enables it for Gradle 6.7, second one for Gradle 6.6. We'll eventually drop the 6.6 variant, but I couldn't risk F-Droid being pissy about early bird Gradle.